README.md

```diff
@@ -68,12 +68,20 @@ The easiest way to install Spyder is:
 
 ### On Windows:
 
-- Or using one of these scientific Python distributions:
+- You can use `pip`:
+
+    pip install spyder
```

Comment:
> I want our users to see it as the first entry :-)

Is it because your work is sponsored by Continuum or there is another reason? Sorry if I am too blunt. I am ok with leaving it as-is, but I feel like it needs a better explanation, because explanations like "because I want it this way" sound very awkward from here. =) There is nothing wrong with sponsored development as long as it doesn't disrupt the development.

My vision to resist the change is that if people are introduced to Python through Spyder using official tools like `pip`, they would be motivated to help with improving those tools as well as core codebase and infrastructure, rather than helping with alternatives that already got their market share by fixing usability bugs independently.

I still bear the hope to make Spyder a replacement for IDLE.

BTW, you said

> The main reason is that we're now creating wheels for Spyder that pull all our dependencies correctly. 

Where are those wheels? I can't see them in https://github.com/spyder-ide/spyder/releases or in AppVeyor builds.